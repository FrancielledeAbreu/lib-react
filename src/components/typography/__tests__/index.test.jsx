import React from 'react'
import Typography from '../index'
import renderer from 'react-test-renderer';

it("renders correctly", () => {
    const tree = renderer.create(<Typography value="" label/>).toJSON()

    expect(tree).toMatchSnapshot()
})