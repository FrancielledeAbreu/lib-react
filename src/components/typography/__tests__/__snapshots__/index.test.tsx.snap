// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdnylx jIjVkI"
>
  <h6 />
</div>
`;
